volatility3/framework/plugins/linux/hidden_modules.py

```diff
@@ -3,7 +3,8 @@
 #
 import logging
 from typing import List, Set, Tuple, Iterable
-from volatility3.framework import renderers, interfaces, exceptions, objects
+import volatility3.framework.symbols.linux.utilities.modules as linux_utilities_modules
```

Comment:
```suggestion
from volatility3.framework.symbols.linux.utilities import modules as linux_utilities_modules
```
Just a stylistic point and has no impact on the code, but I kinda think it's easier to stick with the "always import from" and then do the renaming after?